dataframely/_base_schema.py

```diff
@@ -137,6 +137,29 @@ def __new__(
                         f"which are not in the schema: {missing_list}."
                     )
 
+        # 3) Check that all members are non-pathological (i.e., user errors).
+        for attr, value in {
+            k: v for k, v in namespace.items() if not k.startswith("__")
+        }.items():
```

Comment:
```suggestion
        for attr, value in namespace.items():
            if attr.startswith("__"):
                continue
```

I'd prefer this because 1. I find it easier to read and 2. it avoids an unnecessary second loop pass

Reply:
I'm just gonna commit this change :)